packages/title/src/Title.js

```diff
@@ -2,7 +2,11 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { TextBase } from '@hixme-ui/text'
 
-export const TitleBase = TextBase.withComponent('h1')
+export const TitleBase = TextBase.withComponent('h1').extend`
```

Comment:
Cool, didn't know you could do this.

Reply:
yeah, I'm really impressed with how composable their components are.